selenium_webdriver/src/test/java/tests/sauceDemoTests/SauceDemoMainPageURLAfterSuccessfulLoginTest.java

```diff
@@ -0,0 +1,23 @@
+package tests.sauceDemoTests;
+
+import model.User;
+import org.testng.annotations.Test;
+import pageobject.sauceDemoPages.SauceDemoLoginPage;
+import service.UserCreator;
+import tests.BaseTest;
+
+import static org.testng.Assert.assertTrue;
+
+public class SauceDemoMainPageURLAfterSuccessfulLoginTest extends BaseTest {
+    @Test
+    public void sauceDemoMainPageURLAfterSuccessfulLoginCheck() {
+        User testUser = UserCreator.withCredentialsFromProperty();
+        String sauceDemoMainPageURL = new SauceDemoLoginPage(driver)
+                .openSauceDemoLoginPage()
+                .loginToSauceDemoMainPage(testUser)
+                .getPageURL();
+
+        log.info(String.format("Sauce Demo Main Page Title: %s", sauceDemoMainPageURL));
```

Comment:
not sure you need to log this. Let's discuss)

Reply:
Fixed